inc/commondbtm.class.php

```diff
@@ -5477,4 +5477,21 @@ protected function computeFriendlyName() {
       }
       return '';
    }
+
+   /**
+    * Retrieve an item from the database
+    *
+    * @param integer $ID ID of the item to get
+    *
+    * @return boolean true if succeed else false
+   */
+   public static function getById(int $id) {
+      $item = new static();
+
+      if (!$item->getFromDB($id)) {
+         return false;
+      }
+
+      return $item;
+   }
```

Comment:
Not sure if this ok for you but this is very useful when you need a object reference for a single use action like a condition:

```php
if (Item::getById($y)->xxx()) {
    ...
}
```
instead of 
```php
$item = new (x);
$item->getFromDB($y);
if ($item->xxx()) {
    ...
}
```

Reply:
Ok for me

Reply:
:+1: